src/node/psbt.cpp

```diff
@@ -31,6 +32,10 @@ PSBTAnalysis AnalyzePSBT(PartiallySignedTransaction psbtx)
         // Check for a UTXO
         CTxOut utxo;
         if (psbtx.GetInputUTXO(utxo, i)) {
+            if (!MoneyRange(in_amt) || !MoneyRange(utxo.nValue) || !MoneyRange(in_amt + utxo.nValue)) {
+                result.Reset();
+                return result;
```

Comment:
e769f25031a63c378f2efeb957b90a72a8524a5e

Why not `return {};` instead of `Reset`? Same below, and then drop `PSBTAnalysis::Reset()`.

Reply:
Removed Reset